wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/Reactions/MessageReactionsCellDescription.swift

```diff
@@ -44,7 +44,11 @@ final class MessageReactionsCellDescription: ConversationMessageCellDescription
                 performReaction: nil
             )
         }.sorted {
-            return $0.count > $1.count
+            if $0.count == $1.count {
+                return $0.emoji.name < $1.emoji.name
```

Comment:
I think it would be better to use lexicographical ordering rather than strictly alphabetical ordering, because otherwise all words starting with a capital would precede words without capitals.

Reply:
Is there a test covering the sorting?

Reply:
> I think it would be better to use lexicographical ordering rather than strictly alphabetical ordering, because otherwise all words starting with a capital would precede words without capitals.

Here we're talking about the reactions **below the message**. We don't capitalize any letter so I don't think that would be a problem. 

Regarding the reactions in **Message Details** we capitalize the first letter of all reaction names in every section so that's by default. I don't think we will have a problem there either. Please let me know if you think otherwise.

Reply:
> Is there a test covering the sorting?

I don't think we have a test for that. @johnxnguyen but I can add one.

Reply:
can you use `.sortedByCountThenName()` here?

Reply:
@KaterinaWire no!

Reply:
With code from my other comment you could do something like:

```swift
let reactions = [MessageReactionMetadata] = message.sortedReactions().map { emoji, users in
    MessageReactionMetadata(
        emoji: emoji,
        count: Int(users.count),
        ...
    )
}
```